docs/source/transducer.rst

```diff
@@ -0,0 +1,21 @@
+.. role:: hidden
+    :class: hidden-section
+
+torchaudio.prototype.transducer
+===============================
+
+.. currentmodule:: torchaudio.prototype.transducer
```

Comment:
Did you verify the doc builds? Since prototype module has to be imported separately from `import torchaudio`, I think you need to add `import torchaudio.prototype` to doc config.

Reply:
Yes, the compilation looks good to me. Do you see something missing from the preview in the description?

Reply:
If it compiles, then that's okay. Let's see if the automated push also works.